portal/src/views/projects/ProjectForm.vue

```diff
@@ -245,8 +246,12 @@ export default Vue.extend({
     },
     methods: {
         onTagsFocus(e) {
-            console.log("z", e);
-            document.querySelector('.js-tagsPlaceholder').classList.toggle('focused');
+            document.querySelector('.js-tagsPlaceholder').classList.add('focused');
+        },
+        onTagsBlur(e) {
+           // if (!this.$v.form.tags || this.$v.form.tags.length > 0) {
+                document.querySelector('.js-tagsPlaceholder').classList.remove('focused');
```

Comment:
I'm not a huge fan of using document.querySelector here to modify the classes. Would it be possible to change this to something more idiomatic vue?

Reply:
I see you already made the change. Thanks!
Will keep in mind in the future.